lib/ES/Util.pm

```diff
@@ -725,9 +740,31 @@ sub timestamp {
 }
 
 #===================================
-sub build_docs_js {
+sub start_web_resources_watcher {
 #===================================
-    run '/node_modules/parcel/bin/cli.js', 'build', 'resources/web/docs_js/index.js', '/node_modules', '-d', 'resources/web', '-o', 'docs.js', '--experimental-scope-hoisting';
+    my $parcel_pid = fork;
+    return $parcel_pid if $parcel_pid;
+
+    close STDIN;
+    open( STDIN, "</dev/null" );
+    exec( qw(/node_modules/parcel/bin/cli.js serve
+             --public-url /guide/static/
+             --hmr-port 8001
+             -d /tmp/parcel/
+             resources/web/docs_js/index.js resources/web/styles.pcss) );
```

Comment:
I think if you add `resources/web/docs_js/__tests__/**/* resources/web/docs_js/__tests__/*` here it'll make testing awesome.

Reply:
Haywood and I talked about this off line and I ended up fixing it by improving the Makefile.